examples/flipper/lib.rs

```diff
@@ -67,12 +67,12 @@ mod tests {
 
     #[test]
     fn flipping() -> Result<(), Box<dyn Error>> {
-        let init_value = Session::new(Some(transcoder()))?
-            .deploy(bytes(), "new", &["true"], vec![])?
-            .call("flip", &[])?
-            .call("flip", &[])?
-            .call("flip", &[])?
-            .call("get", &[])?
+        let init_value = Session::new(Some(Rc::new(transcoder())))?
```

Comment:
This probably deserves its own wrapper to hide the gory details :)

Reply:
I moved them to the `transcoder` function - I hope that it will be soon hidden by the ink e2e framework